src/genie/libs/parser/iosxe/tests/ShowGnxiState/cli/equal/golden_output1_state_down_output.txt

```diff
@@ -0,0 +1,3 @@
+State            Status
+--------------------------------
+Disabled         Down
```

Comment:
This is the full output of the command? Nothing else?

Reply:
Nothing else.